src/Tools/Github/GitMergeBot/Program.cs

```diff
@@ -83,7 +83,13 @@ private Program(Options options)
             }
 
             var newBranchName = await MakePrBranch(_options.SourceUser, _options.RepoName, remoteIntoBranch, newBranchPrefix);
-            await SubmitPullRequest(newBranchName);
+            var pullRequest = await SubmitPullRequest(newBranchName);
+
+            // pullRequest could be null if we are running in debug mode.
+            if (pullRequest != null)
+            {
+                await CommentOnIssue(_options.SourceUser, _options.RepoName, pullRequest.Number, "@dotnet-bot test vsi please");
```

Comment:
Nit: `@dotnet-bot` isn't necessary; the regex that triggers the tests is basically ".*test (vsi|unit|etc.) please.*", but I can also argue for the bot mention to make it clear to anybody not familiar with our Jenkins setup that this message doesn't get parsed by a human.